### PR TITLE
Buffs slide barrel shotgun

### PR DESCRIPTION
**Pull request by Humonitarian**
## About The Pull Request

Increases the damage of the slide barrel shotgun

## Why It's Good For The Game

Much needed buff on a suprisingly expensive, yet worthless weapon

## Changelog
:cl:
balance: makeshift slide barrel shotgun buffed
/:cl:

### CODE REVIEW

**Comment by SirRichardFrancis** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up recoil and make all these guns impossible to shoot, thirdly, one and possibly more gun recoil changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

image

**Comment by MLGTASTICa** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up recoil and make all these guns impossible to shoot, thirdly, one and possibly more gun recoil changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

File: code/modules/projectiles/guns/projectile/shotgun/handmade.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
 	w_class = ITEM_SIZE_NORMAL
 	force = WEAPON_FORCE_PAINFUL
-	damage_multiplier = 0.7
+	damage_multiplier = 1.4 // 3 gun parts and only 1 shell capacity, needs this much to be viable - compare to regulator
```
Comment (Iglpigl) [inline code/modules/projectiles/guns/projectile/shotgun/handmade.dm:17]: Direct violation of contributing guidelines.

**Comment by ValoTheValo** [comment]
> "compare to regulator", yeah sure dude im gonna compare a pipe bomb to a hellfire missile.

clearly my peice of literal trash should be on the level of a advanced combat shotgun

**Review by Firefox13 — CHANGES_REQUESTED** [review]
This it too much of a buff

File: code/modules/projectiles/guns/projectile/shotgun/handmade.dm (hunk 1 above)
Comment (Humonitarian) [inline code/modules/projectiles/guns/projectile/shotgun/handmade.dm:17]:
```suggestion
	damage_multiplier = 1.1 // 3 gun parts and only 1 shell capacity, needs this much to be viable - compare to regulator
```
I will now suggest changes to myself to bypass the fact I'm on mobile.

File: code/modules/projectiles/guns/projectile/shotgun/handmade.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 	matter = list(MATERIAL_STEEL = 20, MATERIAL_WOOD = 10)
 	w_class = ITEM_SIZE_NORMAL
 	force = WEAPON_FORCE_PAINFUL
-	damage_multiplier = 0.7
+	damage_multiplier = 1.1 // 3 gun parts and only 1 shell capacity, needs this much to be viable - compare to regulator
```
Comment (TheShown911) [inline code/modules/projectiles/guns/projectile/shotgun/handmade.dm:17]:
```suggestion
	damage_multiplier = 1.1 // 3 gun parts and only 1 shell capacity, needs this much to be viable - compare to other trash guns
```

**Review by TheShown911 — APPROVED** [review]

**Review by Firefox13 — APPROVED** [review]

**Pull request merged**